part-5/import-edit-observed-data.md

```diff
@@ -218,7 +220,72 @@ On the top-right part of the window, one can see the path of the selected source
 
 ### Mapping panel
 
-The left panel of the window displays the mapping of the imported data column to the time, measurement, error values and to the meta data of the observed data sets. The initial mapping is performed automatically upon selection of the file and identification of the format, but it can be overwritten by adjusting the entries. 
+The left panel of the window displays the mapping of the imported data column to the time, measurement, error values and to the meta data of the observed data sets. The initial mapping is performed automatically upon selection of the file and identification of the format, but it can be overwritten by adjusting the entries. This initial mapping recognizes the settings automatically if the data is structured properly. The discovery of columns occurs as follows:
+
+1. Mapping columns (Time, Measurement, Error) are resolved. If the data contains any column with numerical data and a header starting with the mapping name, the column is used for such a mapping. The search is not case sensitive. The following column headers will be recognized as the "Measurement" mapping for example:
```

Comment:
`Mapping columns (Time, Measurement, Error)...`

I would try to find another name for these columns. Maybe "Target **data columns**"?
`Target data columns (Time, Measurement, Error) are mapped.`

Alternatively: just `Time, Measurement and Error columns are mapped.`